CONTRIBUTING.md

```diff
@@ -20,50 +24,54 @@ We are very happy that you consider implementing algorithms and data structures
 ### Making Changes
 
 #### Code
+
 - Please use the directory structure of the repository.
-- File extension for code should be *.h *.cpp.
+- File extension for code should be `*.hpp` `*.cpp`.
```

Comment:
`*.h` is also fine; no need to remove that.

Reply:
AFAIK, if we use `.h` in C++, it will report an error.

Reply:
are you sure? `https://github.com/TheAlgorithms/C-Plus-Plus/tree/master/numerical_methods` this folder has `.h` files and all is fine.

Reply:
Whoops, my bad. Added `*.h` format. 😄